conversion_proxy/src/lib.rs

```diff
@@ -181,6 +213,48 @@ impl ConversionProxy {
         }
     }
 
+    pub fn set_feed_payer(&mut self) {
+        let signer_id = env::predecessor_account_id();
+        if self.owner_id == signer_id {
+            self.feed_payer =
+                Self::get_uuid(env::signer_account_pk()).expect("ERR_OWNER_PK_LENGTH");
+        } else {
+            panic!("ERR_PERMISSION");
+        }
+    }
```

Comment:
Is there a case where the `feed_payer` should be a separate account from the `owner`?

Reply:
I don't know, for now the feed we use is subsidized by Switchboard and given that it's near/USD on Near, they'll probably keep it this way

Reply:
Okay :+1: